File: unity-integration/Assets/GameClustering/Connection.cs
namespace GameClustering
{
    public class Connection
    {
        public string Type { set; get; }
        public long ConnectionId { set; get; }
        public int SessionId { set; get; }
        public string Host { set; get; }
        public int Port { set; get; }
        public bool Secured { set; get; }
    }
}

File: modules/protocol/src/main/java/com/icodesoftware/Connection.java
package com.icodesoftware;

public interface Connection extends Configurable {


    String UDP = "udp";

    String type();
    void type(String type);

    int timeout();
    void timeout(int timeout);

    String serverId();
    void serverId(String serverId);

    boolean secured();
    void secured(boolean secured);

    String host();
    void host(String host);

    int port();
    void port(int port);

}
